src/components/ModalPostVote.vue

```diff
@@ -64,68 +63,69 @@ onMounted(() => {
 
           <template v-else>
             <h3 v-text="$t('proposal.postVoteModal.defaultTitle')" />
-            <p class="italic" v-text="$t('proposal.postVoteModal.tips.1')" />
+            <p class="italic" v-text="'Thank you for your participation!'" />
```

Comment:
Not really related but added this to postvote modal:
<img width="537" alt="image" src="https://github.com/snapshot-labs/snapshot/assets/51686767/a1e82720-3aba-4bd3-99e9-878c4385c98c">

Reply:
Maybe better to update postVoteModal.tips.1 no?

Reply:
Also italic look bit weird, better to keep it normal, but not too important